test-schema.json

```diff
@@ -27,6 +27,20 @@
                 "type": "array",
                 "items": { "$ref": "#/$defs/test" },
                 "minItems": 1
+            },
+            "specification":{
+                "description": "Location of the test case in the specification",
+                "type": "object",
```

Comment:
I suspect an array is more correct here actually , which would allow referencing multiple locations, perhaps even within one spec.

(Or if we want to get fancy at the expense of making it harder to parse we could do `oneOf: [{items: spec}, spec]` and allow either a spec or else an array of specs. But always requiring an array seems fine to me personally.)

Reply:
Concluding this, tell me which will be more readable:
1. `"specifications": [ {"core": ["6.7.1", "7.4.3"] }, { "validation": ["12.2.1", "15.4.2"] } ]`
2.  `"specifications": [ { "core": "6.7.1" }, { "core": "7.4.3" }, { "validation": "12.2.1" }, { "validation": "15.4.2" } ]`
3. `"specifications": { {"core": ["6.7.1", "7.4.3"] }, { "validation": ["12.2.1", "15.4.2"] } }`

and which one is better to be choosen?

Reply:
Concluding this, the instance should look like this: ?
`"specifications": [ { "core": "6.7.1", "quote": "Lorem" }, { "core": "7.4.3" }, { "validation": "12.2.1" }, { "validation": "15.4.2" } ]`

Reply:
Always having an array is easier to handle in some languages.

Reply:
> Concluding this, which of them is better to be chosen:

I'd go with number 1 personally.

> Always having an array is easier to handle in some languages.

Makes sense.

Reply:
great